semeio/jobs/correlated_observations_scaling/scaled_matrix.py

```diff
@@ -47,20 +48,20 @@ def get_scaling_factor(self, events):
         primary components and the number of observations.
         """
         data_matrix = self.get_data_matrix()
-        nr_components, singular_values = self._get_nr_primary_components(
+        nr_components, singular_values = DataMatrix.get_nr_primary_components(
             data_matrix, threshold=events.threshold
         )
         nr_observations = data_matrix.shape[1]
 
-        print("Scaling factor calculated from {}".format(events.keys))
+        logging.info("Scaling factor calculated from {}".format(events.keys))
 
-        return self._calculate_scaling_factor(nr_observations, nr_components)
+        return DataMatrix.calculate_scaling_factor(nr_observations, nr_components)
 
     def _get_data(self):
         return self.data[~self.data.index.isin(["OBS", "STD"])]
 
     @staticmethod
-    def _get_nr_primary_components(data_matrix, threshold):
+    def get_nr_primary_components(data_matrix, threshold):
```

Comment:
I don't really see why this should be a static method 🤔

Reply:
Passing in `self` would be more in line with `get_scaling_factor` and others.

Reply:
Also, `get_nr_primary_components`, `calculate_scaling_factor` and `std_normalization` is not the most coherent API I've ever seen 😂 Should we try to align a bit?